cr.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  Netguru iOS code review task
+//
+
+class paymentViewController: UIViewController {
```

Comment:
Arranging code in groups using "// MARK: - " will make it better and more easy to maintain and read